backend/engine/plugins/trufflehog/type_normalization.py

```diff
@@ -31,6 +32,9 @@ def get_type_normalization_table() -> dict[str, str]:
     for name in _SLACK_TYPES:
         table[name] = "slack"
 
+    for name in _URLAUTH_TYPES:
+        table[name] = "urlauth"
+
```

Comment:
This detector type was missed in https://github.com/WarnerMedia/artemis/pull/271